test/unit/persistent-file.test.js

```diff
@@ -15,14 +15,9 @@ const file = new PersistentFile({
 
 test('PersistentFile#toJSON()', () => {
   const obj = file.toJSON();
-  const len = Object.keys(obj).length;
 
-  expect(1024).toBe(obj.size);
   expect('/tmp/cat.png').toBe(obj.path);
```

Comment:
interesting that that all of these were reverse haha... let's fix them, they should be `expect(obj.path).toBe('/tmp/cat.png')`